Framework/Algorithms/test/FitPeaksTest.h

```diff
@@ -943,7 +943,7 @@ class FitPeaksTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_NOTHING(fitpeaks.setProperty("StopWorkspaceIndex", 1));
     TS_ASSERT_THROWS_NOTHING(fitpeaks.setProperty("PeakFunction", "Gaussian"));
     TS_ASSERT_THROWS_NOTHING(fitpeaks.setProperty("PeakCentersWorkspace", peak_center_ws_name));
-    TS_ASSERT_THROWS_NOTHING(fitpeaks.setProperty("FitPeakWindowWorkspace", fit_window_ws_name))
+    TS_ASSERT_THROWS_NOTHING(fitpeaks.setProperty("FitPeakWindowWorkspace", fit_window_ws_name));
```

Comment:
It doesn't look like there is a test that checks the algorithm for a non-default `MinimumSignalToNoiseRatio!=0` - is it possible to add a check for this where some peaks are excluded using this check?

Reply:
Yes, I am going to add one.